features/src/main/feature/camel-features.xml

```diff
@@ -302,6 +302,12 @@
         <bundle dependency='true'>mvn:org.asteriskjava/asterisk-java/${asterisk-java-version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-asterisk/${project.version}</bundle>
     </feature>
+    <feature name='camel-atmosphere-websocket' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-servlet</feature>
+        <feature prerequisite="true">spifly</feature>
```

Comment:
This is to resolve
`[camel-atmosphere-websocket/4.5.0.SNAPSHOT] osgi.wiring.package; filter:="(&(osgi.wiring.package=org.atmosphere.websocket)(version>=3.0.0)(!(version>=4.0.0)))" [caused by: Unable to resolve org.atmosphere.runtime/3.0.4: missing requirement [org.atmosphere.runtime/3.0.4] osgi.extender; filter:="(osgi.extender=osgi.serviceloader.processor)"]]]
`
same as in `camel-openstack`

Reply:
Surprisingly it compiles without that

Reply:
My bad, it is indeed required